### ISSUE TITLE
binance.pro stream methods does not use defaultType option

### ISSUE THREAD

**Issue by sytranvn** [ccxt/ccxt#24179]
### Operating System

MacOS

### Programming Languages

Python

### CCXT Version

4.4.26

### Description

stream methods does not use defaultType option when set to future

### Code

```py
import asyncio
import ccxt.pro as ccxt  # noqa: E402


async def example():
    binance = ccxt.binance({"options": {"defaultType": "future"}})
    while True:
        tickers = await binance.watch_bids_asks()
        print(tickers)

    await binance.close()


asyncio.run(example())
```

This raises error
```
ccxt.base.errors.ArgumentsRequired: binance watchBidsAsks() requires symbols for self channel for spot markets
```

**Comment by sytranvn** [comment]
Fixed by #24183

**Comment by carlosmiei** [comment]
@sytranvn Can you please upgrade your ccxt version and try again? It should be working now
